scripts/install_vs.ps1

```diff
@@ -0,0 +1,70 @@
+<#
+.SYNOPSIS
+    Installs or updates Visual Studio on a local developer machine
+.PARAMETER Update
+    Update VS to latest version instead of modifying the installation to include new workloads.
+.PARAMETER Quiet
+    Whether to run installer in the background
+#>
+[CmdletBinding(DefaultParameterSetName = 'Default')]
+param(
+    [switch]$Update,
+    [switch]$Quiet
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 1
+
+$intermedateDir = "$PSScriptRoot\obj"
+mkdir $intermedateDir -ErrorAction Ignore | Out-Null
+
+$bootstrapper = "$intermedateDir\vs_enterprise1.exe"
+Invoke-WebRequest -Uri 'https://aka.ms/vs/15/pre/vs_enterprise.exe' -OutFile $bootstrapper
+
+$vsJson = "$PSScriptRoot\VsRequirements\vs.json"
+# no backslashes - this breaks the installer
+$vsInstallPath = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\aspnetci\Enterprise"
```

Comment:
This should probably use the default VS 2017 path.

Reply:
whoops!